extensions/AbuseFilter/AbuseFilter.hooks.php

```diff
@@ -100,7 +100,9 @@ public static function onAbortMove( $oldTitle, $newTitle, $user, &$error, $reaso
 		return $filter_result == '' || $filter_result === true;
 	}
 
-	public static function onArticleDelete( &$article, &$user, &$reason, &$error ) {
+	public static function onArticleDelete(
+		WikiPage $article, User $user, $reason, &$error
```

Comment:
I can see that $user is not used here, so it's possible that some people pass `null` here. If that's the case this method will crash if you make the param typed.